src/OrleansCodeGenerator/CodeGeneratorCommon.cs

```diff
@@ -87,9 +89,13 @@ public static byte[] CompileAssembly(GeneratedSyntax generatedSyntax, string ass
                     .AddSyntaxTrees(code.SyntaxTree)
                     .AddReferences(assemblies)
                     .WithOptions(options);
+            MemoryStream symbolStream = null;
             using (var stream = new MemoryStream())
+#if DEBUG
+            using (symbolStream = new MemoryStream())
+#endif
```

Comment:
Thanks, fixed: I'll emit symbols if any input assembly was built in debug mode & I've undone the behavioral change.